packages/components/src/utils/constants.ts

```diff
@@ -17,5 +17,3 @@ const constants = {
       'https://images.ctfassets.net/tofsejyzyo24/7cOHx0p0J1gUCFMPcoXnFg/73144735e3db345b101509bf707a56b1/Duplex.png',
   },
 };
-
-export default constants;
```

Comment:
was this change to make the constants a named export a depcruiser recommendation or something else? I'm not against it just curious.

Reply:
I thought that it's related but it turned out it isn't ^^ I can revert but it's also my personal recommendation to not use default exports and probably most of the coding guidelines I saw recommend the same. A default export can and will be renamed often on import which makes it hard to find all occurrences of a certain function or component later on.